doc/scss/custom.scss

```diff
@@ -190,3 +190,54 @@ div.sk-text-image-grid-small {
 div.sk-text-image-grid-large {
   @include sk-text-image-grid(100px);
 }
+
+/* Style of the docsearch modal window */
+
+// In the following, we mainly reuse the background color and font-color from the
+// pydata-sphinx-theme. We also adjust slightly the font-size to be wider and in line
+// with the sidebar design.
+:root {
+  --docsearch-modal-width: 768px;
+  --docsearch-modal-height: 700px;
```

Comment:
I wondering if we should do something similar for the width?

Reply:
Nope because horizontally margin is auto (this automatically middle-aligned), and there is already the constraint that the width does not exceed full screen width.

Reply:
OK I see.